widget/embedded/src/components/Quote/Quote.styles.ts

```diff
@@ -162,6 +164,24 @@ export const Chains = styled(Collapsible.Trigger, {
   '& div:nth-child(1)': { display: 'flex' },
 });
 
+export const ChainImageContainer = styled('div', {
+  width: '18px',
+  height: '18px',
+  borderRadius: '100%',
+  border: '1.5px transparent solid',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  variants: {
+    state: {
+      error: {
+        borderColor: '$error500',
+      },
```

Comment:
I like the idea of red border for step with the erorr, but i think in blockchain images row, both icons for the from and destination should be red. Or something like that.
Current state is not very meaningful for me. But i'm not sure how to change it :thinking: 
Please also add a dot (.) at the end of both errors sentences.(Please decrease your amount [here])

![Screenshot from 2023-11-18 15-01-35](https://github.com/rango-exchange/rango-client/assets/99180934/2c75dfc0-53c5-4c6b-a3d8-94219d2410ea)